sunpy/map/mapcube.py

```diff
@@ -52,6 +56,10 @@ class MapCube(object):
     def __init__(self, *args, **kwargs):
         """Creates a new Map instance"""
 
+        # Renaming mapcube functionality to mapsequence
+        warnings.warn("Deprecated in favor of MapSequence. MapSequence has the same functionality as MapCube.",
```

Comment:
The downside of this warning is that if you do `Map(cube=True)` you will get the warning from the factory and this warning.